tests/Unit/NumericalAlgorithms/LinearSolver/ConjugateGradient/Test_ResidualMonitorActions.cpp

```diff
@@ -36,6 +37,8 @@ struct UpdateOperand;
 }  // namespace cg_detail
 }  // namespace LinearSolver
 
+using namespace ResidualMonitorActionsTestHelpers;
```

Comment:
just put the whole file from here to the end inside the namespace instead

Reply:
In other tests we do `using namespace` for helper functions, see e.g. `Test_ReductionObserver.cpp:38`.

Reply:
okay